source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py

```diff
@@ -95,39 +94,6 @@ def get_filter(self, items):
         return " and ".join(filter_list)
 
 
-class add_address_poi_id(object):
-
-    def __init__(self, resource):
-        self.resource = resource
-
-    def __call__(self, f):
-        @wraps(f)
-        def wrapper(*args, **kwargs):
-            objects = f(*args, **kwargs)
-
-            def add_id(objects, region, type_=None):
-                if isinstance(objects, (list, tuple)):
-                    for item in objects:
-                        add_id(item, region, type_)
-                elif hasattr(objects, 'keys'):
-                    for v in objects.keys():
-                        add_id(objects[v], region, v)
-                    if 'address' == type_ and 'coord' in objects:
-                        lon = objects['coord']['lon']
-                        lat = objects['coord']['lat']
-                        objects['id'] = lon + ';' + lat
-                    if 'embedded_type' in objects.keys() and\
-                        (objects['embedded_type'] == 'address' or
-                         objects['embedded_type'] == 'poi' or
-                         objects['embedded_type'] == 'administrative_region') and\
-                         objects['embedded_type'] in objects:
-                        objects["id"] = objects[objects['embedded_type']]["id"]
```

Comment:
In practice, that's a modification, not a creation. I think it's created in fields, generic_type_admin or the like.